### PR TITLE
Search fix

### PR DESCRIPTION
**Pull request by bgrieder**
---

### Fixed

- search of the simple API

### Removed

---

### CODE REVIEW

File: tests/findex/findex.test.ts [hunk 1]
```diff
@@ -19,27 +18,101 @@ import {
 import { USERS } from "../data/users"
 import { expect, test } from "@jest/globals"
 import { createClient } from "redis"
+import { hexEncode } from "../../src/utils/utils"
+import { randomBytes } from "crypto"
+import sqlite3 = require("sqlite3")
```
Comment (ThibaudDauce) [inline tests/findex/findex.test.ts:23]: Can you use `import` here?

Comment (bgrieder) [reply tests/findex/findex.test.ts:23]: yes `import..require` is a special form to accommodate CommonJS

Comment (bgrieder) [reply tests/findex/findex.test.ts:23]: we have all those in `src/utils/utils.ts` for leb128 to fix as well

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 
 ---
 
+## [3.2.2] - 2022-10-19
```
Comment (Manuthor) [inline CHANGELOG.md:7]: we haven't publish the 3.2.1 yet

File: tests/findex/findex.test.ts [hunk 3]
```diff
@@ -93,7 +166,7 @@ test("in memory", async () => {
 })
 
 test("SQLite", async () => {
-  const sqlite3 = require("sqlite3").verbose()
+  // const sqlite3 = require("sqlite3").verbose()
```
Comment (Manuthor) [inline tests/findex/findex.test.ts:169]: remove this?

File: tests/findex/findex.test.ts (hunk 1 above)
Comment (bgrieder) [reply tests/findex/findex.test.ts:23]: another form is `import * as sqlite3 from "sqlite3"` ... but there are subtle differences....

From the mouth of the boss: https://stackoverflow.com/a/35706271/1263942

File: CHANGELOG.md (hunk 2 above)
Comment (bgrieder) [reply CHANGELOG.md:7]: packages.json on main is at 3.2.1 !

File: tests/findex/findex.test.ts (hunk 3 above)
Comment (bgrieder) [reply tests/findex/findex.test.ts:169]: yes

File: tests/findex/findex.test.ts (hunk 1 above)
Comment (ThibaudDauce) [reply tests/findex/findex.test.ts:23]: I think `import sqlite3 from 'sqlite3'` was working on my machine but maybe I'm confused with something else…
Concerning leb128, it's fixed on my branch wasm

File: CHANGELOG.md (hunk 2 above)
Comment (Manuthor) [reply CHANGELOG.md:7]: yes we have just prepared the changelog this morning

Comment (bgrieder) [reply CHANGELOG.md:7]: Do we need to publish 3.2.1 ? why not go direct to 3.2.2 ? Or you want me to change to 3.2.1 ?

Comment (Manuthor) [reply CHANGELOG.md:7]: yes please, use the 3.2.1

**Pull request merged**